src/models/db/conditionFulfillment.js

```diff
@@ -0,0 +1,31 @@
+/* stub class added for body parsing middleware in app.js */
+'use strict'
+
+const Model = require('five-bells-shared').Model
+const PersistentModelMixin = require('five-bells-shared').PersistentModelMixin
+const JsonField = require('sequelize-json')
+const validator = require('../../services/validator')
+const Sequelize = require('sequelize')
+const sequelize = require('../../services/db')
+const Transfer = require('./transfer').Transfer
+
+class ConditionFulfillment extends Model {
+  /*
+   * This will be expanded when we fully decouple fulfillments from transfers
+   */
+}
+
+PersistentModelMixin(ConditionFulfillment, sequelize, {
+  id: { type: Sequelize.INTEGER, primaryKey: true, autoIncrement: true },
+  transfer_id: Sequelize.UUID,
+  type: Sequelize.ENUM('execute', 'cancel'),
```

Comment:
So this is something that gets stored in the database, but is not supposed to be part of the request? Will it cause any problems if the client puts it in the request?